ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/statementofmeans/StatementOfMeansMapper.java

```diff
@@ -260,7 +260,7 @@ public StatementOfMeans from(CCDStatementOfMeans ccdStatementOfMeans) {
             .priorityDebts(priorityDebts)
             .partner(livingPartnerMapper.from(ccdStatementOfMeans.getLivingPartner()))
             .disability(ccdStatementOfMeans.getDisabilityStatus())
-            .carer(ccdStatementOfMeans.getCarer().toBoolean())
+            .carer(ccdStatementOfMeans.getCarer() != null && ccdStatementOfMeans.getCarer().toBoolean())
```

Comment:
should we add toBoolean to handle null?

Reply:
i thought same - could we return false from getCarer()...I suppose we would be populating in final json with false instead of no property

Reply:
It is actually going to populate to false instead of no property.